README.md

```diff
@@ -348,10 +347,21 @@
   ```
 
 * <a name="true-in-cond"></a>
-  Use `true` as the always-match condition of the `cond` special form.
+  If you want to add always-match condition to the `cond` special form then always use `true`.
```

Comment:
Personally, I am 👎 to this change as I have never experienced writing a `cond` that *doesn't* need an always-match condition. 😕